Documenter error

As mentioned in https://github.com/JuliaReach/ReachabilityAnalysis.jl/pull/756#issuecomment-1875406583,  `Documenter` and `Literate` have a new incompatibility (possibly related to Julia v1.10?).

```julia
[ Info: ExpandTemplates: expanding markdown templates.
ERROR: LoadError: AssertionError: length(exs) == 2 * length(results)
Stacktrace:
  [1] parseblock(code::String, doc::Documenter.Document, file::String; skip::Int64, keywords::Bool, raise::Bool, linenumbernode::LineNumberNode)
    @ Documenter ~/.julia/packages/Documenter/1HwWe/src/utilities/utilities.jl:167
```

--- a/examples/LotkaVolterraTangential/LotkaVolterraTangential
+++ b/examples/LotkaVolterraTangential/LotkaVolterraTangential
@@ -223,9 +223,9 @@ validation = []
 final_area = []
 intersect_time = []
 
-# ----------------------------------------
-#  Case 1:
-# ----------------------------------------
+
+# ## Case 1
+
 ε_ext = 1e-4
 prob = lotka_volterra(; nsplit=4, ε_ext=ε_ext)
 alg = TMJets(abstol=1e-14, orderT=7, orderQ=1)
@@ -259,9 +259,7 @@ SUITE[model][cases[1]] = @benchmarkable solve($prob,
                   disjointness_method = BoxEnclosure())
 
 
-# ==============================================================================
-# Execute benchmarks and save benchmark results
-# ==============================================================================
+# ## Execute benchmarks and save benchmark results
 
 # tune parameters
 tune!(SUITE)
@@ -287,9 +285,8 @@ for (i, c) in enumerate(cases)
         " $(final_area[i]), $(intersect_time[i])\n")
 end
 
-# ==============================================================================
-# Create plots
-# ==============================================================================
+
+# ## Create plots
 
 fig = plot()
 

--- a/examples/Quadrotor/Quadrotor.jl
+++ b/examples/Quadrotor/Quadrotor.jl
@@ -73,7 +73,7 @@ using ReachabilityAnalysis: is_intersection_empty
 
 # ## Reachability settings
 #
-#The task is to change the height from ``0``~[m] to ``1``~[m] within ``5``~[s].
+# The task is to change the height from ``0``~[m] to ``1``~[m] within ``5``~[s].
 # A goal region ``[0.98,1.02]`` of the height ``x_3`` has to be reached within
 # ``5``~[s] and the height has to stay below ``1.4`` for all times. After
 # ``1``~[s] the height should stay above ``0.9``~[m]. The initial value for the
@@ -218,21 +218,19 @@ end
 
 cases = ["Δ=0.1", "Δ=0.4", "Δ=0.8"];
 
-# ----------------------------------------
-#  Case 1: smaller uncertainty
-# ----------------------------------------
+# ### Case 1: smaller uncertainty
+
 Wpos = 0.1
 Wvel = 0.1
 prob = quadrotor(; project_reachset=false, Wpos=Wpos, Wvel=Wvel)
 alg = TMJets(; abstol=1e-7, orderT=5, orderQ=1, adaptive=false)
 
-# Warm-up run
+# warm-up run
 sol1 = solve(prob; tspan=Tspan, alg=alg);
 solz1 = overapproximate(sol1, Zonotope);
 
-# ----------------------------------------
-# Case 2: intermediate uncertainty
-# ----------------------------------------
+# ### Case 2: intermediate uncertainty
+
 Wpos = 0.4
 Wvel = 0.4
 prob = quadrotor(; project_reachset=false, Wpos=Wpos, Wvel=Wvel)
@@ -243,18 +241,17 @@ sol2 = solve(prob; tspan=Tspan, alg=alg);
 solz2 = overapproximate(sol2, Zonotope);
 
 # verify that specification holds
-#property = quad_property(solz2)
-#println("Validate property, case $(cases[2]) : $(property)")
+# property = quad_property(solz2)
+# println("Validate property, case $(cases[2]) : $(property)")
 
-# ----------------------------------------
-# Case 3: large uncertainty
-# ----------------------------------------
+# ### Case 3: large uncertainty
+
 Wpos = 0.8
 Wvel = 0.8
 prob = quadrotor(; project_reachset=false, Wpos=Wpos, Wvel=Wvel)
 alg = TMJets(; abstol=1e-7, orderT=5, orderQ=1, adaptive=false)
 
-# Warm-up run
+# warm-up run
 sol3 = solve(prob; tspan=Tspan, alg=alg);
 solz3 = overapproximate(sol3, Zonotope);
 

--- a/test/models/generated/Quadrotor.jl
+++ b/test/models/generated/Quadrotor.jl
@@ -1,8 +1,6 @@
 using ReachabilityAnalysis
 using ReachabilityAnalysis: is_intersection_empty
 
-#The task is to change the height from ``0``~[m] to ``1``~[m] within ``5``~[s].
-
 const g = 9.81           # gravity constant in m/s^2
 const R = 0.1            # radius of center mass in m
 const l = 0.5            # distance of motors to center mass in m
@@ -144,9 +142,6 @@ alg = TMJets(; abstol=1e-7, orderT=5, orderQ=1, adaptive=false)
 
 sol2 = solve(prob; tspan=Tspan, alg=alg);
 solz2 = overapproximate(sol2, Zonotope);
-
-#property = quad_property(solz2)
-#println("Validate property, case $(cases[2]) : $(property)")
 
 Wpos = 0.8
 Wvel = 0.8

